src/components/ui/drawers/tasks/SetupTaskCreationDrawer.tsx

```diff
@@ -23,28 +24,46 @@ export const SetupTaskCreationDrawer = ({
   task,
   editOpen,
   setEditOpen,
+  onUpsertTask,
 }: ISetupTaskCreationDrawer) => {
   const [confirmOpen, setConfirmOpen] = useState(false);
 
   const formControls = useForm<TTaskSchema>({
     resolver: zodResolver(task ? taskUpdateSchema : taskCreationSchema),
     mode: 'onBlur',
-    defaultValues: {
-      categoryId: category?.id,
-      title: task?.title || '',
-      description: task?.description || '',
-    },
+    defaultValues: task
+      ? {
+          categoryId: category?.id,
+          title: task.title || '',
+          description: task.description || '',
+        }
+      : {
+          categoryId: category?.id,
+          title: '',
+          description: '',
+        },
```

Comment:
I believe the previous code was more readable, so could you revert it to the earlier version?

```    defaultValues: {
      categoryId: category?.id,
      title: task?.title || '',
      description: task?.description || '',
    },```